README.rst

```diff
@@ -38,15 +38,15 @@
 .. _Downloads: https://pepy.tech/project/matplotlib/month
 
 .. |NUMFocus| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-.. _NUMFocus: http://www.numfocus.org
+.. _NUMFocus: https://www.numfocus.org
 
-.. image:: doc/_static/logo2.png
+.. image:: https://matplotlib.org/_static/logo2.png
```

Comment:
Why do you link this one to matplotlib.org, but the following one to github.com/matplotlib?

Reply:
And, would it be possible to use the svg logo?

Reply:
The latter was added directly to the repo, but wasn't in the docs site as I think it wasn't updated yet. Since it's now there, I'll link it instead because the shorter URL looks better in the unrendered file.

The image appears to be cached on PyPI's side, so let me see if I can find some docs on supported file types.

Reply:
I see SVG in the readme renderer for Warehouse, so I believe it should work and have updated the URL.

Reply:
No strong opinion on the size, but just checked on mobile. If it‘s not prohibitively large on a Large screen desktop, leave it otherwise scale it down. The air motivation for SVG is that it links good on HiDPI screens.